default-views/auth/sharing.hbs

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>{{title}}</title>
+  <!-- Bootstrap CSS and Theme for demo purposes -->
+  <link rel="stylesheet" href="/common/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/common/css/solid.css">
+</head>
+<body>
+<div class="container title">
+  <h1>Authorize {{app_origin}} to access your Pod?</h1>
+  <p>Solid allows you to precisely choose what other people and apps can read and write in a Pod. This version of the authorization user interface (node-solid-server V5.1.1) only supports the toggle of global access permissions to all of the data in your Pod.</p>
```

Comment:
Not sure it is a good idea to write a version number in the text, it has to be updated by hand, right?

Reply:
Good point. But I don't see an easy way to grab it from the package.json and insert it here. I'll change it to 5.1, because hopefully 5.2 or at least 5.3 will have granular access scopes and we can update or remove this text again already.